.nuxt/google-analytics.js

```diff
@@ -2,6 +2,7 @@ import Vue from 'vue'
 import VueAnalytics from 'vue-analytics'
 
 export default async function ({ app: { router } }) {
-  const moduleOptions = {"id":"UA-65309624-3"}
-  Vue.use(VueAnalytics, Object.assign({ router }, moduleOptions))
+  const options = {"dev":true,"debug":{"sendHitTask":false},"id":"UA-65309624-3"}
```

Comment:
**question**: Why do we need `dev:true` here?

Reply:
I think there may be a bug in the build process or (quite likely) I’m doing something wrong. The dev option is set to true when `npm run dev` but is not set back to false when `npm run build`. I took a clean branch off master to demonstrate:  
run dev : https://github.com/VueVixens/website/commit/69c6bf42c9f8f19ce4d7e020e8d9d4194216c99e#diff-b40bbc46e560f2890a638d70a280e011
run build: https://github.com/VueVixens/website/commit/ace87f10e286c9b5c904b060080503cf22899efe#diff-b40bbc46e560f2890a638d70a280e011

Do you know what script is rewriting that file? If this is a bug, I think that the output should be amended there for a long term fix.

For the purpose of this PR, I ran `git checkout master -- .nuxt/google-analytics.js`

Reply:
I think it could be a different version of Nuxt. Not really crucial th :)